operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/source/SyncingEventSource.java

```diff
@@ -0,0 +1,7 @@
+package io.javaoperatorsdk.operator.processing.event.source;
+
+public interface SyncingEventSource {
```

Comment:
Actually, I don't see where that's being used?

Reply:
hmm I though I deleted it, 1 sec

Reply:
deleted.